src/app/users/users.component.html

```diff
@@ -1,12 +1,37 @@
-<div *ngFor="let user of users">
-  <mat-list role="list">
-    <mat-list-item role="listitem">Id: {{ user.id }}</mat-list-item>
-    <mat-list-item role="listitem">User: {{ user.name }}</mat-list-item>
-    <mat-list-item role="listitem">Email: {{ user.email }}</mat-list-item>
-    <mat-list-item role="listitem">Password: {{ user.password }}</mat-list-item>
-    <section>
-      <button mat-raised-button color="primary">Edit User</button>
-      <button mat-raised-button color="warn">Delete User</button>
-    </section>
-  </mat-list>
-</div>
+<br />
+<button mat-raised-button color="accent" routerLink="new">Create User</button>
+<br />
+<br />
+<table mat-table [dataSource]="users" class="mat-elevation-z8">
+  <!-- Name Column -->
+  <ng-container matColumnDef="name">
+    <th mat-header-cell *matHeaderCellDef>Name</th>
+    <td mat-cell *matCellDef="let element" [@EnterLeave]>
+      {{ element.name }}
+    </td>
+  </ng-container>
+
+  <!-- Email Column -->
+  <ng-container matColumnDef="email">
+    <th mat-header-cell *matHeaderCellDef>Email</th>
+    <td mat-cell *matCellDef="let element" [@EnterLeave]>
+      {{ element.email }}
+    </td>
+  </ng-container>
+
+  <!-- Actions Column -->
+  <ng-container matColumnDef="actions">
+    <th mat-header-cell *matHeaderCellDef>Actions</th>
+    <td mat-cell *matCellDef="let element" [@EnterLeave]>
+      <button mat-mini-fab color="primary" (click)="onEdit(element.id)">
+        Edit
```

Comment:
Remove the word edit,  use and Icon from material Icons.

Reply:
done!!